### PR TITLE
Add proxy protocol to configuration

### PR DESCRIPTION
**Pull request by acogoluegnes**
Part of #589

### CODE REVIEW

File: docs/rabbitmq.conf.example [hunk 1]
```diff
@@ -515,6 +525,16 @@
...
 
+## Whether or not to enable proxy protocol support.
+## Once enabled, clients cannot directly connect to the broker
+## anymore. They must connect through a load balancer that sends the
+## proxy protocol header to the broker at connection time.
+## This setting applies only to STOMP clients, other protocols
+## like MQTT or AMQP have their own setting to enable proxy protocol.
+## See the plugins or broker documentation for more information.
+##
+# stomp.proxy_protocol = false
```
Comment (michaelklishin) [inline docs/rabbitmq.conf.example]: Is there a STOMP plugin PR that extends the schema? I cannot find one.

File: docs/rabbitmq.conf.example [hunk 2]
```diff
@@ -578,6 +598,16 @@
...
 
+## Whether or not to enable proxy protocol support.
+## Once enabled, clients cannot directly connect to the broker
+## anymore. They must connect through a load balancer that sends the
+## proxy protocol header to the broker at connection time.
+## This setting applies only to STOMP clients, other protocols
+## like STOMP or AMQP have their own setting to enable proxy protocol.
+## See the plugins or broker documentation for more information.
+##
+# mqtt.proxy_protocol = false
```
Comment (michaelklishin) [inline docs/rabbitmq.conf.example]: Is there an MQTT plugin PR that extends the schema? I cannot find one.

File: docs/rabbitmq.conf.example (hunk 1 above)
Comment (acogoluegnes) [inline docs/rabbitmq.conf.example]: https://github.com/rabbitmq/rabbitmq-stomp/pull/107

File: docs/rabbitmq.conf.example (hunk 2 above)
Comment (acogoluegnes) [inline docs/rabbitmq.conf.example]: https://github.com/rabbitmq/rabbitmq-mqtt/pull/126

**Pull request merged**